workers/scanner.go

```diff
@@ -43,6 +44,19 @@ const (
 )
 
 var (
+	// AlwaysPinThreshold sets a limit on the contract data of the server. If
+	// the server is below that limit, it will repin underpinned files eve if it
```

Comment:
```suggestion
	// the server is below that limit, it will repin underpinned files even if it
```

Reply:
Even the spell check can't save me from those ones. :D